sqldelight-gradle-plugin/src/test/integration-mysql/src/test/kotlin/com/squareup/sqldelight/mysql/integration/MySqlTest.kt

```diff
@@ -3,27 +3,64 @@ package com.squareup.sqldelight.mysql.integration
 import com.google.common.truth.Truth.assertThat
 import com.squareup.sqldelight.sqlite.driver.JdbcDriver
 import java.sql.DriverManager
+import org.junit.AfterClass
 import org.junit.Before
+import org.junit.BeforeClass
 import org.junit.Test
 
 class MySqlTest {
-    val conn = DriverManager.getConnection("jdbc:tc:mysql:///myDb")
-    val driver = object : JdbcDriver() {
-        override fun getConnection() = conn
-    }
-    val database = MyDatabase(driver)
-
     @Before fun before() {
-        MyDatabase.Schema.create(driver)
+        dogQueries.deleteDogs()
     }
 
     @Test fun simpleSelect() {
-        database.dogQueries.insertDog("Tilda", "Pomeranian", true)
-        assertThat(database.dogQueries.selectDogs().executeAsOne())
+        dogQueries.insertDog("Tilda", "Pomeranian", true)
+        assertThat(dogQueries.selectDogs().executeAsOne())
             .isEqualTo(Dog.Impl(
                 name = "Tilda",
                 breed = "Pomeranian",
                 is_good = true
             ))
     }
+
+    @Test fun simpleSelectWithIn() {
+        dogQueries.insertDog("Tilda", "Pomeranian", true)
+        dogQueries.insertDog("Tucker", "Portuguese Water Dog", true)
+        dogQueries.insertDog("Cujo", "Pomeranian", false)
+        dogQueries.insertDog("Buddy", "Pomeranian", true)
+        assertThat(dogQueries.selectDogsByBreedAndNames(
+            breed = "Pomeranian",
+            name = listOf("Tilda", "Buddy")
+        ).executeAsList())
+            .containsExactly(
+                Dog.Impl(
+                    name = "Tilda",
+                    breed = "Pomeranian",
+                    is_good = true
+                ),
+                Dog.Impl(
+                    name = "Buddy",
+                    breed = "Pomeranian",
+                    is_good = true
+                )
+            )
+    }
+
+    companion object {
+        lateinit var dogQueries: DogQueries
+        val conn = DriverManager.getConnection("jdbc:tc:mysql:///myDb")
```

Comment:
im fine with this, you could also just create a fresh db on @Before by changing the name `myDb` here. Its never used anywhere so you could just random int it or something, idk if the normal thing for testing mysql is to use one db that your clean between tests

Reply:
Typically we'd reuse the DB, though I also don't normally use Testcontainers. Let me see what it looks like to do it that way.